code/API_definitions/carrier-billing.yaml

```diff
@@ -825,8 +826,8 @@ components:
             A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.
 
             Additional Considerations:
-            - When phoneNumber is not indicated, it can be inferred from authorization context (i.e. token), otherwise `HTTP 403` will be answered.
-            - When phoneNumber is indicated, authorization context will be consistent, otherwise `HTTP 403` will be answered.
+            - When phoneNumber is not indicated, it can be inferred from authorization context (i.e. token), otherwise `HTTP 422 - MISSING_IDENTIFIER` will be answered.
+            - When phoneNumber is indicated, authorization context will be consistent, otherwise `HTTP 422 - UNNECESSARY_IDENTIFIER` will be answered.
```

Comment:
this statement is not consistent with the line 126, which is stating:

> If the subject can be identified from the access token and the optional `phoneNumber` field is also included in the request, then the server will return an error with the `422 UNNECESSARY_IDENTIFIER` error code. **This will be the case even if the same phone number is identified by these two methods**, as the server is unable to make this comparison.

Reply:
Got the point, I will propose this, to be aligned

            Additional Considerations:
            - Take a look to section `# Identifying the phone number from the access token` regarding the use of this field.